packages/sanity/src/hydration-test/flood-events.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+
+export class FloodEvents extends React.Component {
```

Comment:
Does this name make sense? I understand this component's use is to "flood" all events but it doesn't really describe the contents of the class